common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Work.scala

```diff
@@ -155,7 +155,8 @@ object WorkState {
 
   case class Source(
     sourceIdentifier: SourceIdentifier,
-    sourceModifiedTime: Instant
+    sourceModifiedTime: Instant,
+    internalWorks: List[Work.Visible[Source]] = Nil
```

Comment:
As a minor note: I think this probably means we can deploy directly into the current prod pipeline, because existing Source works will still deserialise correctly?

Reply:
I think so? I can't think of a reason why they would break